dist/View.js

```diff
@@ -0,0 +1,12 @@
+class View{
+    constructor(){
+        this.sourcePlayer = $(`#entry-template`).html()
+        this.template = Handlebars.compile(this.sourcePlayer)
+    }
+    render(newdata){//method : function inside a class called method 
+        $(`#playerContainer`).empty()
```

Comment:
Chain the `empty()` method while appending, like so:

`$(`#playerContainer`).emptry().append(newHtml)`